modules/restore/README.md

```diff
@@ -0,0 +1,52 @@
+# GCP CloudSQL Restore
+
+## Import from GCS Export Dump
+
+This module can be used for [importing Cloud SQL Postgres database](https://cloud.google.com/sql/docs/postgres/import-export/import-export-sql) from a SQL export dump stored in GCS bucket.
+
+This module uses the SQL export dump file timestamp passed as an input parameter to the Workflow to get the exported dumps from GCS. Following are the steps in import workflow:
+
+1. Fetch list of databases from the source database instance (one that the export was created for)
+2. Delete the databases (list from step 1) except system (`postgres` for Postgres and `tempdb` for SQL Server) databases in the database instance that we are going to import databases to
+3. Create the databases (list from step 1) except system databases in the import database instance
+4. Fetch the SQL export file(s) from GCS and import those into the import database instance
+5. The import API call is asynchronous, so the workflow checks the status of the import at regular interval and wait until it finishes
+
+## How to run
```

Comment:
As discussed offline, let's add the cron functionality you mentioned in a follow up.